src/lovekesh_mishra/DivisibleAssignment13.java

```diff
@@ -0,0 +1,19 @@
+package lovekesh_mishra;
+
+class DivisibleAssignment13 {
+
+	void lastNumber(int startindex, int endindex) {
+		int count = 0;
+		for (int num = startindex; num <= endindex; num++) {
```

Comment:
You can iterate loop from end, then you directly get last number and then break the loop, this will decrease your iterations